src/Templates/Take.Blip.Client.ConsoleTemplate/.template.config/template.json

```diff
@@ -9,5 +9,20 @@
         "type": "project"
       },
     "sourceName": "Take.Blip.Client.ConsoleTemplate",
-    "preferNameDirectory": true
+    "preferNameDirectory": true,
+    "symbols": {
+      "Framework": {
+        "type": "parameter",
+        "description": "The target framework for the project.",
+        "datatype": "choice",
+        "choices": [
+          {
+            "choice": "netcoreapp2.0",
+            "description": "Target netcoreapp2.0"
+          }
+        ],
+        "replaces": "netcoreapp2.0",
+        "defaultValue": "netcoreapp2.0"
+      }
+    }
 }
```

Comment:
What is this symbols properties?

Reply:
Symbols are essential parameters here. In this case the Framework parameter has been added so that users can see the target framework that the project uses. Since there is only one choice (the value set in the project), it cannot be changed, it's more for the users info. The Framework parameter is required if you want to show any other parameters in Visual Studio.